### ISSUE TITLE
When querying SQLite using DuckDB, a mapping of column types must be chosen - example?

### ISSUE THREAD

**Issue by bjornasm** [duckdb/duckdb-web#734]
From the docs about [sqlite scanner](https://duckdb.org/docs/extensions/sqlite_scanner) we can read `When querying SQLite using DuckDB, a mapping of column types must be chosen". However there is no example showing that mapping, and the three examples of querying SQLite does not map column types. It would be super nice with a short code snippet showing how it is preferred we map column types.

**Comment by Mause** [comment]
Hey there, by "chosen" here we mean that DuckDB itself chooses types to map SQLite types to. The section following the part you link to specifies how we map the few SQLite types to DuckDB types. You can customize it using the `sqlite_all_varchar` setting, and afterwards cast to custom types, if you prefer

**Comment by Alex-Monahan** [comment]
Good question! Let me rephrase that a little bit. Does this clarify things @bjornasm? #737

**Comment by bjornasm** [comment]
Thank you so much @Alex-Monahan  - yes! Sorry for being so late with this reply. All in all, thank you for being so active here, also thank you @Mause  .
